### PR TITLE
Add Iris package

### PR DESCRIPTION
**Pull request by bjlittle**
I'd love to get `iris` signed-up to the python3statement!

### CODE REVIEW

File: _sections/30-projects.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/metpy.png)MetPy](https://unidata.github.io/MetPy)
 - [![](assets/nengo.png)Nengo](https://www.nengo.ai/)
 - [![](assets/pylast.png)pyLast](https://github.com/pylast/pylast) <!-- url:https://github.com/pylast/pylast sg:314 -->
+- [![](assets/iris.png)iris](https://scitools.org.uk/iris/docs/latest/) <!-- url:https://github.com/SciTools/iris -->
```
Comment (bjlittle) [inline _sections/30-projects.md]: Not quite sure about the `sg:XXX`, what should be added here?

**Comment by Carreau** [comment]
Don't worry about those, one of us will run a script to count the
"stargazers" (sg:) it's use to reorder the list every now and then if the
star count have change widely.

I can do that once I'm not on my phone.

File: _sections/30-projects.md (hunk 1 above)
Comment (bjlittle) [reply _sections/30-projects.md]: @Carreau Thanks for the explanation for `sg`.... I've now already added the stargazers count for `iris` :smile:

**Comment by takluyver** [comment]
Thanks! Have you already compressed the logo? If not, run `scripts/squash-images.sh` on it to make it as small as possible.

Have you discussed this with the other maintainers of iris? We try to ensure that no-one is surprised by seeing 'their' project on the statement.

**Comment by Carreau** [comment]
(rebased to fix conflicts).

**Comment by bjlittle** [comment]
@takluyver Thanks. 

I'm the lead developer of `iris` at the Met Office, and I can confirm that we already have a core dev quorum agreement with the proposed plan to move to Python3.

We are planning to include this python3statement as part of our public announcement to the wider community on our plans for `iris`. We think it will add more gravitas to our announcement and also highlight the awesome work that you guys are doing to push package developers to commit to Python3 and inform the community in a transparent way.

Cheers

**Comment by takluyver** [comment]
Thanks @bjlittle :-)

**Comment by bjlittle** [comment]
@takluyver and @Carreau awesome thanks for your help.

Just checked https://python3statement.org/#sections40-timeline, but all the timelines appear to be missing. Did this PR break it on master? Or has something else happened?

**Comment by takluyver** [comment]
Ah, I think you might have missed out some commas in the JSON. I should have checked that before merging. :wink: 

Can I ask you to do another PR for that?

**Pull request merged**